### ISSUE TITLE
Add term Hexagonal collagen network of basement membrane

### ISSUE THREAD

**Issue by DaiqingChen** [geneontology/go-ontology#29402]
* **Suggested term label:**
Hexagonal collagen network of basement membrane

* **Definition**
Supramolecular structure formed by trimers of collagen VIII and found in the basement membrane.

* **Reference, in format PMID:#######**
Collagen type VIII trimers form a hexagonal structure: PMID: 21421911
Collagen type VIII trimers are found in basement membranes: PMID: 2376131; PMID: 3047147; PMID: 31387942

* **Gene product name and ID to be annotated to this term**
annotations of collagen type VIII trimer (GO:0005591)
(COL8A1, COL8A2)

* **Parent term(s)**
**part_of** Collagenous component of basement membrane(NTR #29472) 
**is_a** Collagen network (GO:0098645)

* **Children terms (if applicable)** 
collagen type VIII trimer (GO:0005591)

* **Synonyms (please specify, EXACT, BROAD, NARROW or RELATED)**
None

* **Cross-references**

* **Any other information**
**See slides 1, 3, and 7:** https://docs.google.com/presentation/d/1v4o53iJN2Ywo1qi3L8ANZg-p6uCSVaGi/edit?usp=sharing&ouid=109267399050361021326&rtpof=true&sd=true
---
**Slide screenshots:**

<img width="1482" alt="Image" src="https://github.com/user-attachments/assets/b924ac4e-2eba-4df7-a20a-12641a788e34" />

---

<img width="1667" alt="Image" src="https://github.com/user-attachments/assets/66292e40-385b-4058-9cbe-35228f48248f" />

---

<img width="1485" alt="Image" src="https://github.com/user-attachments/assets/70cd86dc-aa4c-4324-a441-6a330f895923" />

**Comment by pgaudet** [comment]
Hi @DaiqingChen 

Can you please provide a reference (pubmed ID) for this new term?

**Comment by DaiqingChen** [comment]
Yes, just added!

**Comment by Matrisome** [comment]
@pgaudet I added of a few additional PMIDs

**Comment by pgaudet** [comment]
Added 

[interstitial hexagonal collagen network](http://purl.obolibrary.org/obo/GO_0098646)
part_of GO:0005614 ! interstitial matrix

**Comment by pgaudet** [comment]
+[Term]
+id: GO:0140155
+name: hexagonal collagen network of basement membrane
+namespace: cellular_component
+def: "Supramolecular structure formed by trimers of collagen VIII and found in the basement membrane." []
+is_a: GO:0098645 ! collagen network
+relationship: part_of GO:0005604 ! basement membrane
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29402" xsd:anyURI
+created_by: pg
+creation_date: 2025-03-26T06:55:36Z

**Comment by DaiqingChen** [comment]
Found collagen type VIII trimer (GO:0005591) part_of interstitial hexagonal collagen network (GO:0098646). Need to remove this relationship, or should discuss it's localization.
In BM: PMID: 39154854 "Collagen VIII has been identified as a major component of the basement membrane of corneal endothelial cells (EC), known as Descemet’s membrane."
PMID: 3047147 "Immunofluorescence studies with the mAb showed that type VIII collagen was deposited as fibrils in the extracellular matrix of corneal endothelial cells. In the fetal calf, type VIII collagen was absent from basement membranes and was found in a limited number of tissues."

**Comment by Matrisome** [comment]
@DaiqingChen, I agree, we should remove the relationship "collagen type VIII trimer (GO:0005591) part_of interstitial hexagonal collagen network (GO:0098646)" and just have "collagen type VIII trimer (GO:0005591) part_of hexagonal collagen network of basement membrane (GO:0140155). 
@pgaudet can you make this change?
